problemtools/config/languages.yaml

```diff
@@ -147,8 +147,8 @@ java:
     name: 'Java'
     priority: 800
     files: '*.java'
-    compile: '/usr/bin/javac -encoding UTF-8 -sourcepath {path} -d {path} {files}'
-    run: '/usr/bin/java -Dfile.encoding=UTF-8 -XX:+UseSerialGC -Xss64m -Xms{memlim}m -Xmx{memlim}m -cp {path} {mainclass}'
+    compile: '/usr/bin/javac -encoding UTF-8 -sourcepath {path} -d {path} -cp {path}/* {files}'
```

Comment:
What is this change for?

Reply:
It allows the problem to include a .jar, and dropping it in the problem directory during the compile, instead of just lose .java files. By using -cp java will look for them in the specified directory.